centralserver/stats/management/commands/write_ip_locations.py

```diff
@@ -1,222 +0,0 @@
-"""
```

Comment:
Ops, please don't delete this. I actively maintain it (last change 23 days ago) and use it (e.g. [here](https://github.com/fle-internal/ansible-playbooks/blob/1ac311e5d570f8698336323d01b0acf99e44a937/roles/keep/tasks/main.yml#L306).